src/utils/TestOverflowItem.tsx

```diff
@@ -1,28 +1,21 @@
 import { Button, makeStyles } from "@fluentui/react-components";
 import * as React from "react";
-import { useOverflowItem } from "../react/useOverflowItem";
+import { OverflowItem } from "../react/OverflowItem";
 
 const useStyles = makeStyles({
-  container: {
-    display: "flex",
-    paddingLeft: "2px",
-    paddingRight: "2px",
+  button: {
+    marginLeft: "2px",
+    marginRight: "2px",
   },
 });
 
 export const TestOverflowItem: React.FC<TestOverflowItemProps> = (props) => {
-  const ref = useOverflowItem<HTMLDivElement>(
-    props.id,
-    props.priority,
-    props.groupId
-  );
   const styles = useStyles();
 
   return (
-    <div ref={ref} className={styles.container}>
-      <Button>Item {props.id}</Button>
-      {props.children}
```

Comment:
None of the scenarios pass children, so I didn't include them in the test item.  This lets the overflow item have one child which is the button.

Reply:
yeah it was meant to be more 'extensible' then realized that it really wasn't necessary :)